packages/@aws-cdk/aws-servicecatalog/package.json

```diff
@@ -44,6 +44,7 @@
     "pkglint": "pkglint -f",
     "package": "cdk-package",
     "awslint": "cdk-awslint",
+    "eslint": "eslint ./lib/*.ts ./lib/private/*.ts ./test/*.ts --fix",
```

Comment:
What's this?

Reply:
artifact from dev branches where we needed to disable a lot of linting until things were working, removed.